scrapers/DesperateAmateurs.yml

```
@@ -0,0 +1,35 @@
+name: "DesperateAmateurs"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - desperateamateurs.com/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="title_bar"]/text()
```

the title selector needs to  be restricted a bit 
it matches the `individual clips` and `updates` also if you check the log
```suggestion
        selector: (//div[@class="title_bar"])[1]
```
should be ok